librocksdb-sys/build.rs

```diff
@@ -357,9 +355,40 @@ fn update_submodules() {
     }
 }
 
+fn update_submodules() {
+    let program = "git";
+    let dir = "../";
+    let args = ["submodule", "update", "--init"];
+    run_command(program, dir, &args);
+}
+
+fn cherry_pick_commits(commits: &Vec<&str>) {
+    for commit in commits {
+        let program = "git";
+        let dir = "rocksdb/";
+        let args = ["cherry-pick", &commit];
+        run_command(program, dir, &args);
+    }
+}
+
+fn apply_patches(patches: &Vec<&str>) {
+    for patch in patches {
+        let program = "git";
+        let dir = "rocksdb/";
+        let args = ["apply", &patch];
```

Comment:
```suggestion
        let args = ["apply", patch];
```

I haven't tried running it but I don't think the additional reference is needed.